k8s/prometheus-federation/deployments/prometheus.yml

```diff
@@ -154,8 +154,6 @@ spec:
                 "--http-source=https://storage.googleapis.com/operator-{{GCLOUD_PROJECT}}/prometheus/snmp-targets/snmpexporter.json",
                 "--http-target=/targets/script-targets/ndt_e2e.json",
                 "--http-source=https://storage.googleapis.com/operator-{{GCLOUD_PROJECT}}/prometheus/script-targets/ndt_e2e.json",
-                "--http-target=/targets/script-targets/ndt_queue.json",
```

Comment:
Please also remove the targets for legacy node-exporter, ssh806, rsyncd, mobiperf, and any others now obsolete.

Reply:
Done.